_posts/2011-10-06-step-by-step-guide.markdown

```diff
@@ -8,24 +8,24 @@ tags:
 <a name="SetUp"></a>
 ## Getting Set Up
 
-Fork documenting-ruby's fork of Ruby [on github](https://github.com/documenting-ruby/ruby).
+Fork the [ruby repository on GitHub](https://github.com/ruby/ruby).
 
 Clone your new repo:
 {% highlight bash %}
-git clone git@github.com:YOUR_NAME/ruby.git
+git clone git@github.com:ruby/ruby.git
```

Comment:
I think, after fork you need to use `clone git@github.com:YOUR_NAME/ruby.git` WDYT?

Reply:
A suspicion - that upper case word is a magic replacement constant in Github pages. Is this true?

Reply:
Oh sorry, I was confused. I’ll amend this instruction text to be complete.